lib/graphql/types.ts

```diff
@@ -0,0 +1,6 @@
+export type FetchOptions = {
+  first?: number | null;
+  last?: number | null;
+  before?: String | null;
+  after?: String | null;
```

Comment:
Why do we allow `number` and `null` as a type? The `?` already allows the type to be `undefined`.

Reply:
blindly followed `before` and `after`